jetty-jaspi/src/main/config/etc/jaspi-basic.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
```

Comment:
This file should be in `src/main/config/etc/jaspi/jaspi-basic.xml` to agree with the location mentioned in the `jaspi-basic-auth-config-factory.mod` file .